src/client.js

```diff
@@ -139,13 +139,27 @@ Client.prototype.connect = function(args) {
 	var webirc = null;
 	var channels = [];
 
-	if (args.join) {
-		var join = args.join.replace(/\,/g, " ").split(/\s+/g);
-		join.forEach(function(chan) {
-			channels.push(new Chan({
-				name: chan
-			}));
+	if (args.channels) {
+		channels = args.channels.map(function(chan) {
+			if (!chan.name) {
+				return;
```

Comment:
Something like `Channel X for network X for user Y is invalid, ignoring` should do. The message doesn't have to be all that useful, it's just a matter of letting the user that *something* is wrong with their config, so the user isn't there "well, it loads my config fine and all but it doesn't join my channel!". I would be ok even with `Invalid channel in config`, the point really is just to give a hint to the user that an error happened.

Personally I would flat out crash (correctness > working), but that won't make a lot of happy people :P